src/lib/plugin_apis/crypto.api

```diff
@@ -1068,10 +1094,9 @@ BDCryptoLUKSTokenInfo** bd_crypto_luks_token_info (const gchar *device, GError *
 /**
  * bd_crypto_integrity_format:
  * @device: a device to format as integrity
- * @algorithm: integrity algorithm specification (e.g. "crc32c" or "sha256") or %NULL to use the default
+ * @algorithm: integrity algorithm specification (e.g. "crc32c" or "sha256")
```

Comment:
I don't see any change of how `algorithm` is handled (in this commit).

Reply:
That was probably some copy-pasting of docstrings between the C and API files. But we shouldn't allow NULL anyway, I added an extra commit dealing with this.